README.md

```diff
@@ -150,6 +150,15 @@ It is possible to remove the component from the tile using `emptyTile` method:
 this.traverser.emptyTile('details');
 ```
 
+### Traversing cycle
+
+| Step      | How | Details |
+|-------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| before      | `beforeTraverse` observable | By subscribing to `beforeTraverse`, we get a Subject and the requested path. If we send `false` to the subject, the traversing is cancelled. |
+| resolve     | `Resolver` class            | The `Resolver` class takes the requested path and obtain the corresponding object that will become the current context.                      |
+| mark        | `Marker` class              | The `Marker` class takes the context object and returns its type.                                                                            |
+| instanciate | `Traverser` service         | According the context type and the requested view, the traverser finds the corresponding component and instantiates it.                      |
```

Comment:
good but I feel like there's too much `the`. I would remove some of them:
> If we send `false` to the subject, ~the~ traversing is cancelled.

> ~The~ `Resolver` class takes the requested path and obtain ~the~ corresponding object that will become the current context. 

> ~The~ `Marker` class takes the context object and returns its type.

>  According the context type and the requested view, ~the~ traverser finds the corresponding component and instantiates it

or maybe even

>  According ~the~ context type and ~the~ requested view, ~the~ traverser finds the corresponding component and instantiates it